mtp_noms_ops/templates/security/credits_history_list.html

```diff
@@ -0,0 +1,96 @@
+{% extends 'base.html' %}
```

Comment:
you could probably use the same template as the pending list. it'll be easier to maintain and it's basically the same page but with a different tab being active/current